cli/lib/tasks/unzip.js

```diff
@@ -113,13 +157,20 @@ const unzip = ({ zipFilePath, installDir, progress }) => {
           case 'darwin':
             return unzipWithOsx()
           case 'linux':
+            return unzipWithUnzipTool()
```

Comment:
but will every Linux system include `unzip`?

Reply:
no, but it silently falls back to node.js unzip if it fails